operator_ui/src/lib/featureFlags.ts

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+
+const FeatureFlagKey = 'features'
+
+export enum Feature {
+  CSAKeys = 'csa_keys',
+  FeedsManager = 'feeds_manager',
+}
+
+type FeatureFlags = {
+  [Property in Feature]: boolean
+}
+
+function getFeatureFlags(): FeatureFlags | null {
+  // Retrieve the object from storage
+  var retrievedObject = localStorage.getItem(FeatureFlagKey)
+
+  if (retrievedObject) {
+    return JSON.parse(retrievedObject) as FeatureFlags
+  }
+
+  return null
+}
+
+// useFeature is a hook which returns whether a feature is enabled/disabled.
+//
+// The flags are stored in localstorage. If you want to enable a feature you
+// will have to manually update local storage with a stringified JSON object
+// under the 'features' key.
```

Comment:
It's a little odd to do it this way as, usually, I'd expect feature flags to come from a server, but :man_shrugging:

Reply:
Yeah this is a temporary measure because we just want to hide features for now until it is all ready. I plan to make changes to add a features endpoint on the server and then update this accordingly